### DISCUSSION TITLE
1.35.3

### DISCUSSION THREAD

**Discussion by Koenkk** [Koenkk/zigbee2mqtt#21267] [General]
Feel free to discuss the [v1.35.3](https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.35.3) release here!

**Comment by cyablo** [comment]
1.35.3 somehow broke my WebInterface, running in Python Virtual Environment on RPi:

![image](https://github.com/Koenkk/zigbee2mqtt/assets/3456720/09fa7d5f-1d0d-4d49-81cc-c07ca86dcb95)

Used update.sh, no errors occured during the update process.

**Reply by Matriciel** [reply]
I have the same issue with Firefox, OK with Chrome

**Reply by Nerivec** [reply]
Look like files didn't compile/copy properly. Might want to try to run the update again to overwrite it all (though you'll probably have to get rid of `./dist/.hash` used to detect if Z2M needs rebuilding, since you had no error, I assume it was set to latest, and would prevent rebuilding). You should see `Building Zigbee2MQTT... (initial build)`, if it works.

**Reply by Nerivec** [reply]
@Matriciel Shot in the dark but, you wouldn't have an ad blocker that's messing with Z2M's files? If Chrome doesn't have one, or not the same... Also could be what I mentioned above, except Chrome is reading the missing files from its cache for now, so you haven't seen the issue (yet). _PS: I've never looked into what Z2M frontend caches or not, and it also depends on your browser's settings, so my 2nd suggestion may or may not apply._)

**Reply by Matriciel** [reply]
Solved, it is a problem with cookies ...

**Reply by cyablo** [reply]
Can confirm.

**Reply by pablo-2023** [reply]
On my mobile **HA app for Android** it was the same, but no tydi up of cookies solved the problem. have had to press the day/night dark/light button  nearby the language selection. little bit tricky, because the screen is not really readable

**Reply by dannypdrsn** [reply]
Same problem on HA app for Android

Any fix?

 ![Screenshot_2024-02-07-10-58-22-75_c3a231c25ed346e59462e84656a70e50.jpg](https://github.com/Koenkk/zigbee2mqtt/assets/83969871/1dcf8b78-c62c-4e4d-87dd-f9313405306f)

**Reply by iqubik** [reply]
delete and install can help

**Reply by Masadido** [reply]
Switching from light to dark mode and then back to light solved the issue for me (click on the little sun icon). However still facing the issue that everytime I click on a a button within Z2M, a new pages is opening in the web browser!
It's working fin on Edge but not on Chrome/Brave browsers

**Comment by iqubik** [comment]
confirm. i clear cookies and all work fine

**Comment by iqubik** [comment]
https://github.com/Koenkk/zigbee2mqtt/issues/21158 Fix single_1 instead of single action for various TuYa devices (@Koenkk)
that's not fixed
![Снимок экрана (382)](https://github.com/Koenkk/zigbee2mqtt/assets/22661808/19cd5a2c-6724-49b9-8c80-58b3c8cf2ba3)

**Comment by Dominik-Sikora** [comment]
https://github.com/Koenkk/zigbee-herdsman-converters/issues/7013 Improve SONOFF SNZB-02 reporting intervals: Will it overwrite my custom reporting settings?

**Reply by sjorge** [reply]
Unfortunately it will :( I've wanted a way to store if they were manually configured or not in the database for a while now. Then we could make this re-(configuration) more intelligent. Just never gotten around to it as it's a lot of work.

**Reply by Dominik-Sikora** [reply]
OK, Z2M tried to overwrite all SNZB-02 reporting settings, but in some sensors, the actions finished with errors, and the settings were not changed. I managed to restore all the settings manually. I hope after the next update, the M2Z will not try to do the same ;)

**Comment by ElouanHocineNexelec** [comment]
I've lost the pictures of my Nexelec Open'R devices in my HA addon view, it went back to the default Zigbee logo.
Any idea where it could come from?
Last commit was updating the description, and i saw you updated docgen/utils.ts to take the ' into account, could it come from this?

![image](https://github.com/Koenkk/zigbee2mqtt/assets/153831974/2f6b39f5-31cf-49bd-a270-d647260fadff)

**Reply by ElouanHocineNexelec** [reply]
Fixed : 
Reinstalling and toggling light/dark mode did the trick for me

**Reply by MBRetrofit** [reply]
Started after toggle

**Comment by jcnico** [comment]
It seems that another easy fix to the display issue is to toggle dark/light mode.

**Comment by MBRetrofit** [comment]
At some point the OTA for Xiaomi Temperature sensor LYWSD03MMC started on this new Z2M but still the one's with 3001-0117 and 3001-0111 shows images not currently available Hardware Version 0.

Also in Dashboard the Xiaomi LYWSD03MMC is not present at all anymore.

**Reply by davlaw** [reply]
I had removed external converter lywsd03mmc.js, thinking I had read somewhere it was not needed anymore.  

Put it back and devices are now in the dashboard

**Reply by davlaw** [reply]
Still no joy with ota

**Reply by iqubik** [reply]
![image](https://github.com/Koenkk/zigbee2mqtt/assets/22661808/814a1022-de89-4026-a78a-112d4d33fd84)

**Reply by davlaw** [reply]
this link worked for me, via the local ota index method

https://github.com/devbis/z03mmc/files/14179644/1141-020a-11003001-z03mmc.zigbee.zip

**Reply by davlaw** [reply]
I can concur that current converter leaves dashboard empty

**Comment by robobobo00** [comment]
thank you for conbee fix !

**Comment by dimab0** [comment]
I am now unable to startup at all. Run into this log:

[12:18:34] INFO: Preparing to start...
[12:18:35] INFO: Socat not enabled
[12:18:35] INFO: Zigbee Herdsman debug logging enabled
[12:18:37] INFO: Starting Zigbee2MQTT...

/app/index.js:91
    settings.reRead();
             ^
TypeError: settings.reRead is not a function
    at start (/app/index.js:91:14)

**Comment by major328** [comment]
> Feel free to discuss the [v1.35.3](https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.35.3) release here!

Hello,

After the upgrade I'm not able to successfully add new devices. In log I see the following messages. The device was recognized, the interview completed and a few second later the device leaves the network and the whole process starts again.If ends somehow in all cases the Z2M reports the interview was not completed successfully.

Error messages: 
![image](https://github.com/Koenkk/zigbee2mqtt/assets/47952234/398d4971-8561-452c-9367-9468441c6afd)
